static/docs/commands-reference/remote.md

```diff
@@ -43,11 +43,7 @@ Using DVC with a remote data storage is optional. By default, DVC is configured
 to use a local data storage only (usually `.dvc/cache` directory inside your
 repository), which enables basic DVC usage scenarios out of the box.
 
-[Add](/doc/commands-reference/remote-add),
```

Comment:
why removing this reference?

Reply:
All the links are already on the top of the document. This reference isn't specific to any one commands, its jus a list of all commands again so I changed to simply "`dvc remote` commands".

Reply:
I would keep ... it's just a nicer UI :) also gives a perspective at what's going on in the description

Reply:
OK, reverted.